prefect_aws/credentials.py

```diff
@@ -67,3 +67,61 @@ def get_boto3_session(self) -> boto3.Session:
             profile_name=self.profile_name,
             region_name=self.region_name,
         )
+
+
+class MinIOCredentials(Block):
```

Comment:
Just a random thought, should this be `MinIoCredentials` to be consistent with `AwsCredentials` and `GcpCredentials`?

I like typing `MinIOCredentials` more through, `MinIoCredentials` is a lot more tricky to type, so maybe we can keep it ` MinIOCredentials` (or another random thought, `MinioCredentials`?

Reply:
Since it's technically `MinIO` and it's easier, it makes more sense to keep it as `MinIOCredentials` in my opinion. Only `MinioCredentials` could be an alternative, but I find that to look a little odd.